### PR TITLE
feat: issues

### PR DESCRIPTION
**Pull request by Simperfit**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets |  |
| License | MIT |
| Doc PR |  |

### CODE REVIEW

File: src/Bridge/NelmioApiDoc/Parser/ApiPlatformParser.php [hunk 1]
```diff
@@ -128,6 +128,7 @@ private function parseResource(ResourceMetadata $resourceMetadata, string $resou
 
     private function getGroupsContext(ResourceMetadata $resourceMetadata, string $operationName, bool $isNormalization)
     {
+        $operation = [];
```
Comment (meyerbaptiste) [inline src/Bridge/NelmioApiDoc/Parser/ApiPlatformParser.php]: What do you think about that?
```php
return [
    $groupsContext => [
        'groups' => array_merge(isset($itemOperationAttribute) ? $itemOperationAttribute : [], isset($collectionOperationAttribute) ? $collectionOperationAttribute : []),
    ],
];
```

**Comment by dunglas** [comment]
Can you add a non-regression test please?

**Comment by dunglas** [comment]
What does it fix exactly? I don't get what is changed.

**Comment by Simperfit** [comment]
It was fixing an issue in scrutinizer (where $operation should be defined as an array)

**Comment by dunglas** [comment]
Ok, can you rebase please?

**Comment by Simperfit** [comment]
@dunglas rebased

**Comment by dunglas** [comment]
Thanks @Simperfit

**Pull request merged**